src/components/Modals/SelectPassenger/SelectPassenger.js

```diff
@@ -59,15 +120,100 @@ const TopSection = styled.section`
   padding: 20px 24px 24px 24px;
   font-size: 14px;
   font-weight: 600;
-  border-bottom: none;
 `;
 
-const CloseBtn = styled.button``;
+const CloseBtn = styled.button`
+  position: absolute;
+  top: 20px;
+  right: 20px;
+
+  svg {
+    width: 24px;
+    height: 24px;
+    margin: 0;
+  }
+`;
 
-const Title = styled.h3``;
+const Title = styled.h3`
+  font-size: 14px;
+  font-weight: 600;
+`;
 
 const BottomSection = styled.section``;
 
-const Passengers = styled.div``;
+const Passengers = styled.div`
+  padding: 0 24px 24px 24px;
+`;
 
-const Seats = styled.div``;
+const Passenger = styled.dl`
+  display: flex;
+  justify-content: space-between;
+
+  &:not(:first-of-type) {
+    margin-top: 16px;
+  }
+
+  dt {
+    span {
+      display: block;
```

Comment:
```suggestion
  dt span {
      display: block;
```
이런 식으로 해도 동일하지 않을까요? 가급적 nesting이 적으면 좋습니다

Reply:
숙지하고 있겠습니다, 감사합니다 👀